doc/mgr/administrator.rst

```diff
@@ -44,12 +44,12 @@ Calling module commands
 Where a module implements command line hooks, using the Ceph CLI's
 ``tell`` command to call them like this::
 
-    ceph tell mgr <command | help>
+    ceph tell mgr <command | get_command_descriptions>
```

Comment:
That would be nice.  The "normal" help command (for mon commands) is implemented in the CLI client (see function `do_extended_help`), we could have a similar thing for `tell` -- the client already has the command descriptions for `tell`, as we can see when it does the "no valid command found; 10 closest matches:" output, so implementing help should be simple.